### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by Eitot**
First draft

### CODE REVIEW

File: website/de.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		"support": "Support",
+		"blog": "Blog",
+		"careers": "Jobs",
+		"donate": "Spenden",
+		"mediaKit": "Pressemappe",
+		"threatModel": "Bedrohungsmodell",
+		"logIn": "Anmelden",
+		"signUp": "Konto erstellen",
+		"cta": "Ihr sicheres E-Mail-Konto erstellen",
+		"forSupportInquiries": "Für Support, gehen Sie nach",
```
Comment (FWirtz) [inline website/de.json]: I dont really like "gehen Sie nach", I think "please visit" rather translates to "besuchen Sie bitte"

File: website/de.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+		"privacyPolicy": "Datenschutzerklärung",
+		"termsAndConditions": "Allgemeine Gebrauchsbestimmungen",
+		"transparencyReport": "Transparenzbericht",
+		"pricing": "Preise",
+		"support": "Support",
+		"team":"Team",
+		"feedbackForum": "Feedbackforum"
+	},
+	"home": {
+		"pageTitle": "Sichere E-Mail: ProtonMail ist kostenlose, verschlüsselte E-Mail.",
```
Comment (FWirtz) [inline website/de.json]: would prefer "ProtonMail **bietet** kostenlose, ..."

File: website/de.json [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+		"loading": "Laden…",
```
Comment (FWirtz) [inline website/de.json]: Definitely rather "Lädt..." than "Laden..."

File: website/de.json [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+		"title": "Sichere E-Mail<br>aus der Schweiz",
```
Comment (FWirtz) [inline website/de.json]: dont know about that one...
Maybe "Sichere E-Mail**s**"?

File: website/de.json [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+		"subtitle": "Sichern Sie Ihre Kommunikation mit ProtonMail",
+		"cta": "Verschlüsseltes E-Mail-Konto erstellen",
+		"appTitle": "Die ProtonMail-Apps für mobile Geräte sind nun weltweit erhältlich.",
+		"appSubTitle": "Verschlüsselte E-Mail für Ihr mobiles Gerät",
```
Comment (FWirtz) [inline website/de.json]: see above, maybe rather "E-Mail**s**" than "E-Mail"

File: website/de.json (hunk 1 above)
Comment (Eitot) [inline website/de.json]: I agree.

File: website/de.json (hunk 5 above)
Comment (Eitot) [inline website/de.json]: The original was singular too. I am not sure whether I should change that.

File: website/de.json (hunk 3 above)
Comment (Eitot) [inline website/de.json]: I’ll change it.

File: website/de.json (hunk 4 above)
Comment (Eitot) [inline website/de.json]: In the English source, they use the singular form as well. I was thinking of adding a noun to it, like ‘Sichere E-Mail-Kommunikation’ or like, but not sure.

File: website/de.json (hunk 2 above)
Comment (Eitot) [inline website/de.json]: Sounds better, yes, but that is what the source says. :/

Comment (FWirtz) [inline website/de.json]: true. But that only really works in english, in german its just wrong. I think it wouldnt sound right

File: website/de.json (hunk 5 above)
Comment (FWirtz) [inline website/de.json]: true. See [this](http://www.nytimes.com/2010/05/09/magazine/09FOB-onlanguage-t.html?_r=0) for reference, kinda agree with you though.
Still, in my opinion, it sounds wrong in german.

File: website/de.json [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+		"logIn": "Anmelden",
+		"signUp": "Konto erstellen",
+		"cta": "Ihr sicheres E-Mail-Konto erstellen",
+		"forSupportInquiries": "Für Support, besuchen Sie",
+		"forSecurityDiscussion": "Für Sicherheitsfragen, wenden Sie sich an",
+		"footerCopyright": "&copy; 2016 Proton Technologies AG. Alle Rechte vorbehalten.",
+		"security": "Sicherheit",
+		"imprints": "Impressum",
+		"privacyPolicy": "Datenschutzerklärung",
+		"termsAndConditions": "Allgemeine Gebrauchsbestimmungen",
```
Comment (ChrisSte) [inline website/de.json]: I would suggest "Geschäftsbedingungen" instead of "Allgemeine Gebrauchtsbestimmungen".

File: website/de.json [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+		"chooseLanguage": "Wählen Sie Ihre Sprache",
+		"currentLanguage": "Deutsch",
+		"secureEmail": "Sichere E-Mail",
+		"about": "Über",
+		"security": "Sicherheit",
+		"support": "Support",
+		"blog": "Blog",
+		"careers": "Jobs",
+		"donate": "Spenden",
+		"mediaKit": "Pressemappe",
```
Comment (ChrisSte) [inline website/de.json]: I would suggest just "Presse". Compared other german websites and they just have "Presse" written.

File: website/de.json [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+		"threatModel": "Bedrohungsmodell",
+		"logIn": "Anmelden",
+		"signUp": "Konto erstellen",
+		"cta": "Ihr sicheres E-Mail-Konto erstellen",
+		"forSupportInquiries": "Für Support, besuchen Sie",
```
Comment (ChrisSte) [inline website/de.json]: Not sure about this one, but what would you think about: "Für Unterstützung besuchen Sie bitte" ?

File: website/de.json [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+		"forSecurityDiscussion": "Für Sicherheitsfragen, wenden Sie sich an",
+		"footerCopyright": "&copy; 2016 Proton Technologies AG. Alle Rechte vorbehalten.",
+		"security": "Sicherheit",
+		"imprints": "Impressum",
+		"privacyPolicy": "Datenschutzerklärung",
+		"termsAndConditions": "Allgemeine Gebrauchsbestimmungen",
+		"transparencyReport": "Transparenzbericht",
+		"pricing": "Preise",
+		"support": "Support",
```
Comment (FWirtz) [inline website/de.json]: I've never seen "Unterstützung" anywhere, so I'd prefer "Hilfe" over that one

File: website/de.json (hunk 8 above)
Comment (FWirtz) [inline website/de.json]: Maybe rather "Um Hilfe zu bekommen, besuchen Sie bitte"...? But I dont quite like that as well... :sweat_smile:

File: website/de.json (hunk 2 above)
Comment (Eitot) [inline website/de.json]: It sounds odd in English too, that was my point. Probably good to have some feedback or guidance from ProtonMail.

File: website/de.json [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+		"featE2ETitle": "Ende-zu-Ende-Verschlüsselung",
+		"featE2ESubTitle": "Automatischer E-Mail-Schutz",
+		"featE2EDetails": "Alle E-Mails werden automatisch mit einer Ende-zu-Ende Verschlüsselung gesichert. Das bedeutet, dass selbst wir Ihre E-Mails nicht entschlüsseln können. Demzufolge können wir Ihre E-Mails auch nicht an Dritte weitergeben.",
+		"featAnonTitle": "Anonyme E-Mail",
+		"featAnonSubTitle": "Schützen Sie Ihre Privatsphäre",
+		"featAnonDetails": "Es sind keine personenbezogenen Daten nötig, um Ihr Konto zu erstellen. Standardmäßig protokollieren wir auch nicht ihre IP-Adressen, die Sie identifizieren können. Ihre Privatsphäre ist vorrangig.",
+		"featFreeTitle": "100% kostenlos",
+		"featFreeSubTitle": "Kostenlose, sichere E-Mail",
+		"featFreeDetails": "Wir glauben, dass E-Mail-Datenschutz und -Sicherheit menschliche Grundrechte sind, die jedem zustehen sollten. Die Basiskonten von ProtonMail werden immer kostenlos sein. Sie können das Projekt unterstützen indem Sie <a href='https://protonmail.com/donate'>spenden</a> oder <a href='https://protonmail.com/pricing'>Ihr Konto aufwerten</a>.",
```
Comment (N3tt3rm4n) [inline website/de.json]: "menschliche Grundrechte" or "_grundlegende Menschenrechte_"? In German "menschliche Grundrechte" sounds a little bit odd, and in the English original says "fundamental human rights".

File: website/de.json [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+		"featEasyTitle": "Einfach zu verwenden",
+		"featEasySubTitle": "Unkomplizierte Sicherheit",
+		"featEasyDetails": "ProtonMail kann auf jedem Gerät genutzt werden, ohne zusätzliche Software. ProtonMails sichere E-Mail-Konten sind gänzlich kompatibel mit anderen E-Mail-Anbietern. Sie können ganz normal E-Mails versenden und empfangen.",
+		"featDesignTitle": "Zeitgemäßes Design des Posteingangs",
```
Comment (N3tt3rm4n) [inline website/de.json]: Maybe "Modernes Design" would suit better...

File: website/de.json [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+		"featDesignSubTitle": "Sicherheit und Produktivität",
+		"featDesignDetails": "Der ProtonMail-Posteingang wurde für Produktivität optimiert. Jedes Detail unseres sicheren E-Mail-Dienstes wurde optimiert, um Ihnen beim Lesen, Verwalten und Versenden von E-Mails zu unterstützen.",
```
Comment (N3tt3rm4n) [inline website/de.json]: "um Ihnen beim Lesen, Verwalten und Versenden von E-Mails zu unterstützen" is gramatically wrong. Should be "um Sie [...] zu unterstützen" or "um Ihnen [...] zu helfen. Imho the first is better.

File: website/de.json [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+		"designTitle": "Sicherheit trifft auf zeitgemäßes Design",
```
Comment (N3tt3rm4n) [inline website/de.json]: See above - "zeitgemäß" or "modern"?

File: website/de.json [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+		"designSubTitle": "Unser Posteingang der nächsten Generation wurde entworfen, um Ihnen die Arbeit zu erleichtern. Die Verschlüsselung, die wir zu Ihrem Schutz einsetzen, ist praktisch unsichtbar.",
+		"securityTitle": "Serverschutz",
+		"securityDetails": "ProtonMails Server befinden sich in Europas sicherstem Datenzentrum, unter 1000 Meter hohem Gestein.",
```
Comment (N3tt3rm4n) [inline website/de.json]: "unter 1000 Meter hohem Gestein" sounds odd to me, imho "unter 1000 Metern Gestein" is better.

Comment (Eitot) [inline website/de.json]: I guess I took the source too literally. 😂

File: website/de.json (hunk 12 above)
Comment (Eitot) [inline website/de.json]: You are right, I changed this sentence multiple times and overlooked it.

File: website/de.json (hunk 10 above)
Comment (Eitot) [inline website/de.json]: I changed it.

**Comment by Super-Baleine** [comment]
Hey guys, the translation project's [out there](https://crowdin.com/project/protonmail), that's useless to put anything here 🐳

**Pull request closed without merging**